minio/minio
# add support for configurable replication MRF workers

## Description
add support for configurable replication MRF workers

## Motivation and Context
just like replication workers, allow failed replication
workers to be configurable in situations like DR failures
etc to catch up on replication sooner when DR is back
online.

Signed-off-by: Harshavardhana <harsha@minio.io>

## How to test this PR?
Add DR failure scenarios and see how quickly
MRF workers catch up on the failed operations.

## Types of changes

## Checklist:
